apps/src/redux/isRunning.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * This duck module tracks whether the current app is running or not.
+ */
+var TOGGLE_RUNNING = 'isRunning/TOGGLE_RUNNING';
+
+module.exports.default = function (state, action) {
+  var nextState = state || false;
+
+  if (action.type === TOGGLE_RUNNING) {
+    nextState = action.isRunning;
+  }
+  return nextState;
+};
+
+module.exports.toggleRunning = function (isRunning) {
+  return {
+    type: TOGGLE_RUNNING,
+    isRunning: isRunning
+  };
+};
```

Comment:
Feels a little silly to have this whole module for a single bool. Could be that I can combine it with some similar state at some point.